### DISCUSSION TITLE
Custom CSS/JS not working on GitHub Page

### DISCUSSION THREAD

**Discussion by tahayparker** [community/community#56005] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have a [GH Page](https://tahayparker.github.io/CSProjGuide/) on one of my repos, and my custom css and js is not loading.
tried everything i could, from ensuring https in external css/js to waiting for a couple days.
please help, this website has to go out to my peers in about a week's time.
tia :)

**Comment by airtower-luna** [comment]
The "custom CSS" [linked in your `index.html`](https://github.com/tahayparker/CSProjGuide/blob/302e061a673055413e19efa7c630327077f6300e/index.html#L10) isn't actual CSS, it's SCSS which you'd need to compile to CSS using Sass (or write actual CSS directly). See also: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Organizing#build_systems_for_css

**Comment by tahayparker** [comment]
Thank you, this was the result of following another discussion on github and i ended up doing more harm than good.
@airtower-luna really grateful for your help and support!

**Comment by tahayparker** [comment]
Now my custom cursors and favicons are not appearing on the website :(

**Comment by airtower-luna** [comment, marked as answer]
You have invalid absolute links there. E.g. [for the favicon](https://github.com/tahayparker/CSProjGuide/blob/0412bd2a738d2d17a5f43e9fe296a2b1aa92cfc2/index.html#L19-L20C4):

```html
    <link rel="icon" type="image/png" sizes="32x32" href="/favicons/favicon-32x32.png">
    <link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png">
```

Because of the leading slash the browser will look at `https://tahayparker.github.io/favicons/favicon-32x32.png`, which doesn't exist. You should use relative links, in this specific case removing the leading slash will work.

The problem with the cursor is basically the same, mind that in that case the relative location is relative to the stylesheet.

See: https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#absolute_urls_vs._relative_urls

**Comment by tahayparker** [comment]
i fixed the favicons, but i cannot fix the cursor

**Comment by tahayparker** [comment]
update: fixed it :)
